package.json

```diff
@@ -50,24 +50,19 @@
     "webpack-merge": "^4.0.0"
   },
   "engines": {
-    "node": ">= 4.3 < 5.0.0 || >= 5.10"
+    "node": ">= 4.8 < 5.0.0 || >= 5.10"
   },
   "peerDependencies": {
     "node-sass": "^4.0.0",
-    "webpack": "^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0"
+    "webpack": "^2.0.0 || ^3.0.0 || ^4.0.0"
```

Comment:
Also breaking change :disappointed: Let's add just `^4.0.0`

Reply:
Why is this a breaking change ? Polluting this prop with `alpha`, `beta`, `rc` versions is really not a good idea and most folks should have updated to the latest `webpack@3.0.0` by now I assumed.

Reply:
@michael-ciniawsky someone can use `rc.0` and it can break build, but i agree with you

Reply:
Then it breaks :)